Gemfile

```diff
@@ -59,6 +59,7 @@ gem "sentry-rails"
 gem "sentry-ruby"
 # Pagination
 gem "pagy"
+gem "possessive" 
```

Comment:
Is this for a case that ActiveSupport `.pluralize` doesn't cover?

Reply:
pluralize pluralizes the word, so instead of `Baark's` we'd get `Baarks`

Reply:
Guessing we don’t want to pluralize a user name, we just want to make sure we say either: Paul’s or James’ (not James’s)

Reply:
Makes sense